aspnet/fundamentals/error-handling.rst

```diff
@@ -1,8 +1,181 @@
-.. include:: /../common/stub-topic.txt
+Error Handling
+==============
+By `Steve Smith`_
 
-|stub-icon| Error Handling
-==========================
+When errors occur in your ASP.NET app, you can handle them in a variety of ways, as described in this article.
```

Comment:
We need to be really clear when we're dealing with Exceptions vs status codes. Avoid the word Error from here down.